assets/src/data/content/writers/html.tsx

```diff
@@ -87,7 +89,7 @@ export class HtmlParser implements WriterImpl {
       .reduce((acc, mark) => mark(acc), <>{text}</>);
   }
 
-  private figure(context: WriterContext, attrs: any, content: React.ReactElement) {
+  private captioned_content(context: WriterContext, attrs: any, content: React.ReactElement) {
```

Comment:
we had a `figure` function in here that wasn't the actual figure element. Refactored that to "captioned_content" to hopefully not confuse a future developer